src/styles/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
```

Comment:
do you need to specify it here?

Reply:
Don't understand where else can I specify it?

Reply:
done